automation/UNIX/build-package.sh

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env bash
+
+# Build an installation package for clang.
+
+function resetBeforeExit {
+  set +ue
+  set +o pipefail
+}
+
+function cmdsucceeded {
+  resetBeforeExit
+  exit 0
+}
+
+function cmdfailed {
+  echo "Build installation package failed."
+  resetBeforeExit
+  exit 1
+}
+
+set -ue
+set -o pipefail
+set -x
+
+cd ${LLVM_OBJ_DIR}
+
+if [ "${BUILD_PACKAGE}" == "No" ]; then cmdsucceeded; fi
+
+echo "Building installation package for clang"
+
+if [ "${TEST_TARGET_ARCH}" == "X86" ]; then
```

Comment:
Clang is a cross-compiler by default that can target multiple architectures.  This setting tells clang to only target X86.  I think you want to delete this if statement.

Reply:
I copied that from run-cmake.sh lines 9-13. Does it play a role there or should that be cleared out too?

Reply:
I think run-cmake.sh needs to be updated too.